nef/Utils/Bookmark.swift

```diff
@@ -0,0 +1,54 @@
+//  Copyright © 2019 The nef Authors.
+
+import Foundation
+
+@propertyWrapper
+struct Bookmark {
+    private let key: String
+    private let storage = UserDefaults.standard
+    
+    init(key: String) {
+        self.key = key
+    }
+
+    var wrappedValue: URL? {
+        get {
+            guard let url = retrieveBookmark(), existItem(at: url) else { return nil }
+            openAccessing(bookmark: url)
+            return url
```

Comment:
Is there a problem if the bookmark is never closed?

Reply:
In the Apple documentation we can read:
```
When access to this resource is no longer needed the client must call stopAccessingSecurityScopedResource. Each call to startAccessingSecurityScopedResource must be balanced with a call to stopAccessingSecurityScopedResource (Note: this is not reference counted).
```

This is the reason :)

Reply:
I see, but I am still concerned about not calling it sometimes. It is open here, but one cannot really see where it is being closed. It looks like a fantastic place to use `Bracket` or `Resource`.

Reply:
As we have talked I will work on a proposal using Resource ^^